pulsar_metrics/metrics/utils.py

```diff
@@ -34,36 +34,35 @@ def get_population_percentages(new: pd.Series, reference: pd.Series, binned: boo
 
             return percents
     except Exception as e:
-        print(f"Error while calculating population percentages: {str(e)}")
+        print(f"Error in get_population_percentages() while calculating population percentages: {str(e)}")
 
 
 def kullback_leibler_divergence(new: pd.Series, reference: pd.Series, binned: bool = False):
     """
-    Calculates the Kullback-Leibler divergence
+    Calculate Kullback-Leibler divergence (KL divergence) for samples (new,reference)
     """
 
     percents = get_population_percentages(new, reference, binned)
 
-    kl_div = entropy(percents["new"], percents["ref"])
-
-    return kl_div
+    return entropy(percents["new"], percents["ref"])
 
 
 def population_stability_index(new: pd.Series, reference: pd.Series, binned: bool = False):
```

Comment:
Clarification: If the input type is of pf.Dataframe. Will the same code works? Do you need isinstance() validation before calculation.

Reply:
We can add a validation that new and reference are unidimensional. However this function will be changed in the future